### PR TITLE
optionally hide the progress bar for loading

### PR DESCRIPTION
**Pull request by JasonMWhite**
This PR adds the ability to optionally hide the progress bar when extracting the data, primarily since it makes test output noisy for no benefit.

### CODE REVIEW

File: etl/src/energuide/extractor.py [hunk 1]
```diff
@@ -95,16 +96,19 @@ def _extract_snippets(row: typing.Dict[str, typing.Any]) -> typing.Dict[str, typ
...
 
 
-def extract_data(input_path: str) -> typing.Iterator[typing.Optional[typing.Dict[str, typing.Any]]]:
+def extract_data(input_path: str,
+                 show_progress: bool = False) -> typing.Iterator[typing.Optional[typing.Dict[str, typing.Any]]]:
     validator = cerberus.Validator(INPUT_SCHEMA, purge_unknown=True)
-    for data in _read_csv(input_path):
+    rows = _read_csv(input_path, show_progress)
+
+    for row in rows:
```
Comment (sastels) [inline etl/src/energuide/extractor.py]: just curious, why did you change this to save the generator in `rows` (as opposed to just putting it all inline `for row in _read_csv(input_path, show_progress):`)?

File: etl/src/energuide/cli.py [hunk 2]
```diff
@@ -64,11 +64,12 @@ def load(username: str,
 @main.command()
 @click.option('--infile', required=True)
 @click.option('--outfile', required=True)
-def extract(infile: str, outfile: str) -> None:
+@click.option('--progress/--no-progress', default=True)
```
Comment (sastels) [inline etl/src/energuide/cli.py]: isn't click awesome!

File: etl/src/energuide/extractor.py (hunk 1 above)
Comment (JasonMWhite) [reply etl/src/energuide/extractor.py]: I just found it a little too dense to keep track of, once I also put the `tqdm` progress bar in `_read_csv`.
